reaper/reaper.go

```diff
@@ -90,6 +90,9 @@ func (m *Monitor) Wait(c *exec.Cmd, ec chan runc.Exit) (int, error) {
 			// make sure we flush all IO
 			c.Wait()
 			m.Unsubscribe(ec)
+			if e.Status != 0 {
+				return e.Status, errors.New("unsucessful command")
```

Comment:
I still don't understand why we need an error for non-0 exit status.  Its not an error that we need to handle in our code.

Reply:
but then `runc.Create()` returns success on failure since `Monitor.Wait()` will return `status, nil`

Reply:
I see the problem now.  For the runc calls, we want a non-0 to == a failed execution for runc and stop what we are doing.  but for containers an error does not matter from what as we just report back the exit status. 

Maybe an exit error type, the one from the exec package, so that the caller can decide if they want to ignore it or not?

Reply:
we can't use `ExiitError` unfortunately. `os.ProcessState` fields are private

Reply:
I guess we can remove the `Output` and `CombinedOutput` in the go-runc package and then handle status !=0 in the go-runc lib for all the calls.  This will give us the functionality that we want both in containerd and in go-runc calls

Reply:
`Run` needs removed as well.  The `ProcessMonitor` interface should only have `Start` and `Wait` on it after the changes